docs/how-to-use-minio-server-side-encryption-with-aws-cli.md

```diff
@@ -1,88 +1,94 @@
+
+**Note**: Tools like `aws-cli` or `mc` will display an error if a self-signed TLS certificate is used when trying to upload objects to the server. See [Let's Encrypt](https://letsencrypt.org/) to get a CA-signed TLS certificate. Self-signed certificates should only be used for development, testing or internal usage.
+
+# <a name="use-sse-c-with-aws-cli"></a>Use SSE-C with aws-cli
+
+This section describes how to use server-side encryption with customer-provided encryption (SSE-C) keys via the aws-cli.
+
+## Install the aws-cli 
+You can install the AWS Command Line Interface using the procedure described [here](https://docs.minio.io/docs/aws-cli-with-minio).
+
+## Create a bucket named `my-bucket`
+
+```sh
+aws --no-verify-ssl --endpoint-url https://localhost:9000 s3api create-bucket --bucket my-bucket
+```
+
+## Upload an Object using SSE-C
+
+The following example shows how to upload an object named `my-secret-diary` where the content is the file: `~/my-diary.txt`:
+
+```sh
+aws s3api put-object \
+  --no-verify-ssl \
+  --endpoint-url https://localhost:9000 \
+  --bucket my-bucket --key my-secret-diary \
+  --sse-customer-algorithm AES256 \
+  --sse-customer-key MzJieXRlc2xvbmdzZWNyZXRrZXltdXN0cHJvdmlkZWQ= \
+  --sse-customer-key-md5 7PpPLAK26ONlVUGOWlusfg== \
+  --body ~/my-diary.txt You should use your own encryption key.
+```
+
+In this example, a local Minio server is running on https://localhost:9000 with a self-signed certificate. TLS certificate verification is skipped using: `--no-verify-ssl`. If a Minio server uses a CA-signed certificate, then `--no-verify-ssl` should not be included, otherwise aws-cli would accept any certificate.
+
+
+## Display Object Information
+Specify the correct SSE-C key of an encrypted object to display its metadata:
+
+```sh
+Copy  aws s3api head-object \
   --no-verify-ssl \
   --endpoint-url https://localhost:9000 \
   --bucket my-bucket \
   --key my-secret-diary \
   --sse-customer-algorithm AES256 \
   --sse-customer-key MzJieXRlc2xvbmdzZWNyZXRrZXltdXN0cHJvdmlkZWQ= \
   --sse-customer-key-md5 7PpPLAK26ONlVUGOWlusfg==
-  ```
-
-### 2.3 Download an object
-
-1. Now delete your local copy of `my-diary.txt`:  
-   `rm ~/my-diary.txt` 
-   
-2. You can restore the diary by downloading it from the server:
-   ```
-   aws s3api get-object \
-   --no-verify-ssl \
-   --endpoint-url https://localhost:9000 \
-   --bucket my-bucket \
-   --key my-secret-diary \
-   --sse-customer-algorithm AES256 \
-   --sse-customer-key MzJieXRlc2xvbmdzZWNyZXRrZXltdXN0cHJvdmlkZWQ= \
-   --sse-customer-key-md5 7PpPLAK26ONlVUGOWlusfg== \
-   ~/my-diary.txt
-   ```
-
-   
+```
+
+## Download an Object
+The following examples show how a local copy of a file can be removed and then restored by downloading it from the server:
+
+Delete your local copy of `my-diary.txt`:
+
+```sh
+rm ~/my-diary.txt
+```
+
+Restore the file by downloading it from the server:
+
+```sh
+Copyaws s3api get-object \
```

Comment:
Is Copyaws correct here or should this just be `aws`?

Reply:
@eco-minio Revising to just `aws`.